src/features/timeline/PersonRelationListItem.tsx

```diff
@@ -0,0 +1,77 @@
+import Typography from '@mui/material/Typography';
+import { timeFormat } from 'd3-time-format';
+
+import type { Relation } from '@/features/common/entity.model';
+import { useGetPersonByIdQuery, useGetPlaceByIdQuery } from '@/features/common/intavia-api.service';
+
+const dateFmt = timeFormat('%B %_d, %Y');
+
+interface PersonRelationListItemProps {
+  relation: Relation;
+}
+
+export function PersonRelationListItem(props: PersonRelationListItemProps): JSX.Element {
+  const { relation } = props;
+
+  // pattern: <action> [<other entity name>] [in <place name>] [on <date>]
+  const verb =
+    relation.type === 'beginning' ? 'born' : relation.type === 'end' ? 'died' : relation.type;
+
+  const otherEntityName =
+    relation.targetId === undefined ? '' : <OtherEntityName targetId={relation.targetId} />;
+
+  const placeName = relation.placeId === undefined ? '' : <PlaceName placeId={relation.placeId} />;
+
+  const date =
+    relation.date === undefined ? (
+      ''
+    ) : (
+      <Typography variant="body2" component="span">
+        {' on '}
+        <Typography variant="body2" component="span" sx={{ fontWeight: 'bold' }}>
+          {dateFmt(new Date(relation.date))}
+        </Typography>
+      </Typography>
+    );
+
+  return (
+    <Typography variant="body2" component="li">
+      {verb}
+      {otherEntityName}
+      {placeName}
+      {date}
+    </Typography>
+  );
+}
+
+function OtherEntityName(props: { targetId: string }): JSX.Element {
+  const { targetId } = props;
+
+  const searchResult = useGetPersonByIdQuery({ id: targetId });
```

Comment:
in a followup, let's lift these up to the parent (and wait until we have everything we need to render) to avoid tooltip text shifting when request resolves